bundles/org.openhab.binding.sonnen/src/main/java/org/openhab/binding/sonnen/utilities/Helper.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sonnen.utilities;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link Helper} is a Helper class to overcome Call by value for a Status Description.
+ *
+ *
+ * @author Christian Feininger - Initial contribution
+ */
+@NonNullByDefault
+public class Helper {
```

Comment:
This class looks a little strange to me.
Maybe you could have `serviceCommunication.refreshBatteryConnection` simply returning null if ok or an error message in case of error.

Reply:
Or the method could simply throw an exception and you will use the exception message to set the detailed thing status message.